test/intervals/TestIntervals.jl

```diff
@@ -183,7 +181,9 @@ facts("IntervalCollection") do
 
 
     context("Show") do
-        nullout = open("/dev/null", "w")
+        nullstream = @windows ? "NUL" : "/dev/null"
+        nullout = open(nullstream, "w")
```

Comment:
Here I think you can use the `DevNull` object, which was broken on v0.4.1 but is now available on v0.4.2.